cmd/internal/planetscale_edge_database.go

```diff
@@ -7,7 +7,6 @@ import (
 	"net/http"
 	"strings"
 	"time"
-
```

Comment:
Can you add back the line here to reduce the diff? And in general, these get formatted with an `stdlib` block first etc.